src/test/java/pl/java/spring/web/project/clientappproject/login/controller/UserValidatorMockTest.java

```diff
@@ -0,0 +1,52 @@
+package pl.java.spring.web.project.clientappproject.login.controller;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.validation.BeanPropertyBindingResult;
+import org.springframework.validation.Errors;
+import org.springframework.validation.FieldError;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RunWith(MockitoJUnitRunner.class)
+@SpringBootTest
+public class UserValidatorMockTest {
+
+    @InjectMocks
+    private UserValidator userValidator;
+
+    @Mock
+    private UserService userServiceMock;
+
```

Comment:
It is worth adding here an AOP adnotation like @Before to inject all mocks before.
@Before
    public void setUp() throws Exception {
          MockitoAnnotations.initMocks(this);
    }